### ISSUE TITLE
Error while fecthing extensions. XHR failed

### ISSUE THREAD

**Issue by bagustris** [microsoft/vsmarketplace#961]
Type: <b>Bug</b>

Error while fecthing extensions. XHR failed

VS Code version: Code 1.91.1 (f1e16e1e6214d7c44d078b1f0607b2388f29d729, 2024-07-09T22:08:12.169Z)
OS version: Linux x64 5.15.0-116-generic snap
Modes:

**Comment by Neelima10735584** [comment]
Hi @bagustris,

Thanks for reaching out. If you're using a VPN client or some local firewall, please turn these off and retry. If the error is still persisting, it will help us to investigate, if you can share the network log (command: Developer: Open Logs Folder).

Regards,
VS Marketplace Team

**Comment by bagustris** [comment]
@Neelima10735584 

I don't use vpn. I tried on two different networks, both failed. I tried to uninstall (vscode deb) and then install snap version, both failed. Then I tried several hours later; the problem was gone, and now I can install extensions again. :)

I have identic installation in another machine (v1.19.1 on Ubuntu 20.04), but doesn't face this problem.

Perhaps network problem, but I already tried on two different networks.
